flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Why did it become x86_64 only?

Reply:
no support for arm64

Reply:
thats not what I'm seeing:

![image](https://github.com/flathub/flathub/assets/114441/53f94a49-b117-4528-8860-e46cd7851ac4)

Please remove the restriction.

Reply:
I don't think it makes sense to do that.

According to the author's words:
"Really x86/x86-64 on Linux is really the only WELL supported platform."
https://github.com/eteran/edb-debugger/issues/808

Also, if the program compiles I cannot test the result.

Reply:
but if you don't build it no one can test it either. THere is explicit support for aarch64.

Reply:
In fact the author says no. and it also says it in the code, it is possible that the software does not work well. I asked the developer, I'm waiting for the answer.

Reply:
link?

Reply:
https://github.com/eteran/edb-debugger/issues/808

Reply:
Author here 🙂

The only platform that edb can currently be expected to work in any meaningful way is x86-64 on Linux.

Due to the nature of the project, it has a lot of highly OS and architecture specific code and arm64 has never been in a state that can build, let alone work (so far!). We hope to improve this at some point, but it's challenging given that doing so involves buying new hardware, learning new arch specific details, and a lot of testing.